test/infrastructure/docker/OWNERS

```diff
@@ -1,9 +1,6 @@
 # See the OWNERS docs: http://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-  - sig-cluster-lifecycle-leads
-  - cluster-api-admins
-  - cluster-api-maintainers
   - cluster-api-provider-docker-maintainers
 
 reviewers:
```

Comment:
Do you want to add `cluster-api-reviewers` here as well?

Reply:
This is probably ok to keep separate wdyt? CAPD code is a different beast imo 😄

Reply:
possibly...  But one could likely also say the same for the different providers. I'm fine leaving it as is for now, but I think we'll need to grow the reviewer group over time especially as we actually use the e2e test signal as a blocker for release.